modules/veteran_confirmation/app/services/veteran_confirmation/service.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'mpi/service'
```

Comment:
I don't think this file is being used anywhere?

Reply:
Yeah good catch. I don't think the configuration file is needed either actually.

Reply:
👍